src/util/api/likernft/book/upload.ts

```diff
@@ -0,0 +1,33 @@
+import { Storage } from '@google-cloud/storage';
+import { CACHE_BUCKET } from '../../../../constant';
+
+const storage = new Storage();
+const bucket = storage.bucket(CACHE_BUCKET);
+
+export default async function uploadFile({
```

Comment:
```suggestion
const bucket = storage.bucket(CACHE_BUCKET);

export default async function uploadFileToBookCache({
```

This function always target `storage.bucket(CACHE_BUCKET)`

Reply:
[🎨 Rename uploadFile to uploadFileToBookCache](https://github.com/likecoin/likecoin-api-public/pull/922/commits/f6de801d9e3eaa412a205f2e4149c19fa1768754)